flag.go

```diff
@@ -1171,6 +1188,13 @@ func (f *FlagSet) Parsed() bool {
 	return f.parsed
 }
 
+// SetUnknownFlagsSlice arranges to append any unknown flags to the
+// given slice. This requires ParseErrorsWhitelist.UnknownFlags to be
+// set so that parsing does not abort on the first unknown flag.
+func (f *FlagSet) SetUnknownFlagsSlice(s *[]string) {
```

Comment:
I'm afraid I don't recall what I was thinking last week, let alone 20 months ago ;-)

My motivation for needing this ended up being going away due to other changes needed in the consuming code base and I've also since change jobs. I'm happy if someone else wants to carry this change though.

Reply:
This response sums up my experience trying to contribute to this repo. I've waited two years for review. That's insane.

Reply:
The cobra community would like this (or something like this) so that we can have an api access to unknown flags. Ether way would be fine as long as we can get them!

Reply:
I'm finally merging down a few of these PRs into my fork and I'll update the readme to explain how to override the dependency when using cobra.

Reply:
@jpmcb Okay. I've made some changes to the way this was implemented and [merged it](https://github.com/cornfeedhobo/pflag/pull/1) into [my fork](https://github.com/cornfeedhobo/pflag). I hope that suits your needs, but feel free to open any issues and I'll jump on them.

@ijc Thanks for putting this together and making it easy!

Reply:
Thanks for putting a fork together. However, I think it would be an anti-goal of the cobra project to fracture viper, pflags, and cobra by introducing a fork as a core dependency. Cobra is really just a wrapper for pflags and viper. So ideally, I'd like if this could get some love and be merged

Reply:
@jpmcb I mean, yeah I feel the same way. But this repo currently has a broken test for over 2 years and it has not been fixed. I have 3 pull requests that are 2+ years old and no response. I've pinged @eparis and @therealmitchconnors multiple times without much traction. I'm going to keep my fork going. Everyone can decide what they want.

Reply:
@jpmcb I mean, yeah I feel the same way. But this repo doesn't get the stewardship it needs. I have 3 pull requests that are 2+ years old and no response. One of them fixes a broken test that has gone unnoticed for well over 2 years. Just looking at the commit history, it looks like the only PRs that get touched here are ones that have internal pressure/connections. One shouldn't have to flash their company credentials to get attention in a reasonable time frame.

Reply:
@jpmcb Yeah, I'd like that too, but I don't want to wait another 2 years for a response. No one has an obligation to use my fork, but I'm movin' on :)